internal/cmd/connect/command_plugin_install.go

```diff
@@ -24,31 +24,28 @@ import (
 )
 
 type manifest struct {
-	Name    string `json:"name"`
-	Title   string `json:"title"`
-	Version string `json:"version"`
+	Name    string    `json:"name"`
+	Title   string    `json:"title"`
+	Version string    `json:"version"`
+	Owner   owner     `json:"owner"`
+	Archive archive   `json:"archive"`
+	License []license `json:"license"`
```

Comment:
Plural since it's a list?
```suggestion
	Licenses []license `json:"licenses"`
```